packages/data-api/package.json

```diff
@@ -20,7 +20,6 @@
     "uninstall-mv-refresh": "scripts/uninstallMvRefreshModule.sh",
     "build-analytics-table": "scripts/buildAnalyticsMaterializedView.sh",
     "drop-analytics-table": "scripts/dropAnalyticsMaterializedView.sh",
-    "full-rebuild-analytics-table": "scripts/fullRebuildAnalyticsMaterializedView.sh",
```

Comment:
Decided to remove this script as it was just a combination of 2 other scripts. Instead detailed how to do this process in the README